index.js

```diff
@@ -2,17 +2,44 @@
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
-
 /***** Deliverable 2 *****/
 
+header.style.color = 'darkblue'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+let pImg = document.querySelector("#profile img")
+let pName = document.querySelector("#profile h2")
+let pNickname = document.querySelector("#profile em")
+
+pImg.src = traveler.photo
+pImg.alt = "Raffy Pic"
+pName.innerHTML = traveler.name
+pNickname.innerHTML = traveler.nickname
```

Comment:
`innerHTML` works here, but in general it's better to only use `innerHTML` if you're creating DOM elements, like this:

```js
pName.innerHTML = `<span>traveler.name</span>`
```

For just inserting plain strings of text, using `textContent` is better:

```js
pName.textContent = traveler.name
pNickname.textContent = traveler.nickname
```